axelor-project/src/main/resources/views/Dashboard.xml

```diff
@@ -338,19 +338,34 @@
       <field type="date" name="toDate" title="To Date" required="true"/>
     </search-fields>
     <dashlet height="350" action="project-project-task-action" canSearch="true" colSpan="12"/>
+    <dashlet height="350" action="project-planning-time-dashboard-action" canSearch="true"
+      colSpan="12"/>
```

Comment:
I don't know if it's possible to add it here too 

if="config.app.isApp('employee') && config.app.getApp('project')?.enablePlanification" if-module="axelor-human-resource"